packages/synapse-react-client/src/components/TermsAndConditions/TermsAndConditionsItem.tsx

```diff
@@ -46,28 +90,31 @@ const TermsAndConditionsItem: React.FunctionComponent<
   }
 
   return (
-    <>
-      <span className="terms-icon">{item.icon}</span>
+    <li className={enabled ? 'terms-enabled' : ''}>
+      <span
+        className="terms-icon"
+        dangerouslySetInnerHTML={{ __html: iconFileContent }}
```

Comment:
This is difficult because the table cell lacks multi-line editor (even in the case when the column type is LARGE_TEXT).  Maybe this should change, but I'm not planning to include that change without a lot of discussion with the group